### DISCUSSION TITLE
Is that from the maplibre organisation

### DISCUSSION THREAD

**Discussion by mariusvn** [maplibre/flutter-maplibre-gl#326] [General]
https://pub.dev/packages/maplibre_gl_platform_interface

This sounds weird that a vietmap repo end up as maplibre as author

**Comment by m0nac0** [comment]
Yes, the package was transferred to the official maplibre account.

**Reply by m0nac0** [reply]
However, it is worth noting that the versions on pub.dev are currently broken, see #309
